parser/src/cfg/mod.rs

```diff
@@ -2646,31 +2647,39 @@ fn parse_layers(s: &mut ParsedState, mapped_keys: &mut MappedKeys) -> Result<Int
                     let input = &triplet[0];
                     let mapstr = &triplet[1];
                     let action = &triplet[2];
-                    let input_key = input
-                        .atom(s.vars())
-                        .and_then(str_to_oscode)
-                        .ok_or_else(|| anyhow_expr!(input, "input must be a key name"))?;
-                    mapped_keys.insert(input_key);
-                    if !layer_mapped_keys.insert(input_key) {
-                        bail_expr!(input, "input key must not be repeated within a layer")
-                    }
-                    let mapstrs = ["=", ":", "->", ">>", "maps-to", "→", "🞂"];
-                    mapstr
-                        .atom(s.vars())
-                        .and_then(|s| match mapstrs.contains(&s) {
-                            true => Some(()),
-                            false => None,
-                        })
-                        .ok_or_else(|| {
-                            anyhow_expr!(
-                                mapstr,
-                                "map string must be one of the following strings:\n\
+                    if input.atom(s.vars()).is_some_and(|x| x == "_") {
+                        let action = parse_action(action, s)?;
+                        for (i, _) in s.mapping_order.iter().enumerate() {
+                            layers_cfg[layer_level * 2][0][s.mapping_order[i]] = *action;
+                            layers_cfg[layer_level * 2 + 1][0][s.mapping_order[i]] = *action;
+                        }
+                    } else {
+                        let input_key = input
+                            .atom(s.vars())
+                            .and_then(str_to_oscode)
+                            .ok_or_else(|| anyhow_expr!(input, "input must be a key name"))?;
+                        mapped_keys.insert(input_key);
+                        if !layer_mapped_keys.insert(input_key) {
+                            bail_expr!(input, "input key must not be repeated within a layer")
+                        }
+                        let mapstrs = ["=", ":", "->", ">>", "maps-to", "→", "🞂"];
+                        mapstr
+                            .atom(s.vars())
+                            .and_then(|s| match mapstrs.contains(&s) {
+                                true => Some(()),
+                                false => None,
+                            })
+                            .ok_or_else(|| {
+                                anyhow_expr!(
+                                    mapstr,
+                                    "map string must be one of the following strings:\n\
```

Comment:
looking at the diff on GitHub, you'd think I changed/added all these lines, but if you use `git diff` with [delta](https://github.com/dandavison/delta) you would see a better diff, which show that I only added indentation to these lines, because I added an `if else` block, and put the existing code in the `else` block.

Reply:
If any of you guys are using Refined Github browser extension (or if not, I recommend :)), you can hide whitespaces in diff by clicking this button 
![image](https://github.com/jtroo/kanata/assets/44318430/64315461-dab2-4cd6-aa41-6dd86915ae6a)

Reply:
It's built-in too, but not default:

![image](https://github.com/jtroo/kanata/assets/6634136/e8c4f6cd-0b21-43c8-93ef-e99d73059497)

Reply:
I see, that's nice. I though Refined Github adds this feature, but it only adds a convenience button.